src/doc/trpl/crates-and-modules.md

```diff
@@ -430,7 +388,7 @@ fn main() {
 }
 ```
 
-But it is not idiomatic. This is significantly more likely to introducing a
+This is more likely to introduce a
```

Comment:
Cool. One of the jobs of the book is to inform people about idioms, so I'd like this to stay.